### PR TITLE
add Kucoin Announcements

### PR DESCRIPTION
**Pull request by CyberPunkMetalHead**
* added new option to listen to kucoin_announcements in config.example.yml
* if there are no Binance announcements, the bot will try to return a kucoin_announcement

### CODE REVIEW

File: new_listings_scraper.py [hunk 1]
```diff
@@ -53,19 +55,65 @@ def get_announcement():
     return latest_announcement['data']['catalogs'][0]['articles'][0]['title']
 
 
+def get_kucoin_announcement():
```
Comment (DominicFrei) [inline new_listings_scraper.py:58]: Most of that seems identical to the way we check Binance. Would it make sense to extract some kind of `create_url(exchange)` function?

File: new_listings_scraper.py [hunk 2]
```diff
@@ -53,19 +55,65 @@ def get_announcement():
...
+        kucoin_coin = re.findall('\(([^)]+)', kucoin_announcement)
 
+    found_coin = re.findall('\(([^)]+)', latest_announcement)
     uppers = None
 
+    # returns nothing if it's an old coin or it's not an actual coin listing
     if 'Will List' not in latest_announcement or found_coin[0] == globals.latest_listing or \
             found_coin[0] in previously_found_coins:
-        return None
+
```
Comment (DominicFrei) [inline new_listings_scraper.py:106]: Previously we returned `None` if nothing could be found (on Binance). Here we check KuCoin now but if that does not work, we don't eventually return `None` anymore. Can this lead to problems somewhere else? Is it missing?

**Review by DominicFrei — COMMENTED** [review]
Looks good to me so far. I've added two questions but I don't think they would prevent a merge. :+1:

File: new_listings_scraper.py (hunk 2 above)
Comment (CyberPunkMetalHead) [reply new_listings_scraper.py:106]: I don't think that'll be an issue. uppers is defined as None on line 101 and the function will return uppers on line 126, so I think unless uppers is re-assigned inside the Kucoin logic, the function will still return none.

File: new_listings_scraper.py (hunk 1 above)
Comment (CyberPunkMetalHead) [reply new_listings_scraper.py:58]: Can definitely be consolidated I agree. I'll merge this in for now if all good, and will try and think of a way to reduce the amount of duplicated code.

Comment (DominicFrei) [reply new_listings_scraper.py:58]: Fair enough. :+1:

File: new_listings_scraper.py (hunk 2 above)
Comment (DominicFrei) [reply new_listings_scraper.py:106]: Ah, good point. Should be safe I guess. :+1:

**Review by DominicFrei — APPROVED** [review]

**Pull request merged**